Jenkinsfile

```diff
@@ -61,8 +61,8 @@ def buildManifest = {String manifest, String bitbake_image ->
         sh "mkdir -p archive"
         sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/tmp/deploy/images/ /vagrant/archive/\""
 
-        // Archive the downloads and sstate when we build the master
-        if (env.BRANCH_NAME == "master") {
+        // Archive the downloads and sstate when we build the master in multi pipeline
+        if (env.ARCHIVE_CACHE || env.BRANCH_NAME == "master") {
```

Comment:
Hm, or perhaps we shouldn't archive the cache on a multi pipeline job, just during the nightly builds which can be parametrized?